debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java

```diff
@@ -672,7 +672,7 @@ private Throwable handleRetries(final RetriableException e, final List<Map<Strin
                 startedSuccessfully = true;
             }
             catch (Exception ex) {
-                if (totalRetries >= maxRetries) {
+                if (maxRetries != EmbeddedEngineConfig.DEFAULT_ERROR_MAX_RETRIES && totalRetries >= maxRetries) {
```

Comment:
@jpechane @vjuranek Do you think it is a good idea to reference the const for the default value here? This condition should only yield true when the value is explicitly -1. Imagine that in a future PR the default value is going to be updated, then this line would be invalid and the code misbehaves. wdyt?

Reply:
@rk3rn3r Hi, yes, it is good idea. But the fact is that what is not correct is not the use of the constant but the name of the constant. It is both default and special value and the name should reflect it is the special value, not default value.